### PR TITLE
Add a recipe for Qt4

### PR DESCRIPTION
**Pull request by ccordoba12**
@ocefpaf, @pelson, @jjhelmus, here it is our recipe for Qt4.

For now I'm trying to build it only for Linux.

### CODE REVIEW

File: recipes/qt4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+package:
+  name: qt
+  version: {{ version }}
+
+source:
+  fn: qt-everywhere-opensource-src-{{ version }}.tar.gz
+  url: http://download.qt-project.org/official_releases/qt/4.8/{{ version }}/qt-everywhere-opensource-src-{{ version }}.tar.gz
+  md5: d990ee66bf7ab0c785589776f35ba6ad
+  patches:
```
Comment (pelson) [inline recipes/qt4/meta.yaml]: Definitely interested in each of these patches either having good comments, or a xref to a patch to the repo.

File: recipes/qt4/notes.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+On Windows, for VS 2015 (Python 3.5+), rather drastic modifications are
```
Comment (pelson) [inline recipes/qt4/notes.md]: I guess we can move some of these details into comments in the recipe.

File: recipes/qt4/meta.yaml (hunk 1 above)
Comment (ccordoba12) [inline recipes/qt4/meta.yaml]: I added a comment for all patches

**Comment by ccordoba12** [comment]
@ocefpaf, since this passed please merge. I can improve the recipe in the feedstock ;-)

**Comment by ocefpaf** [comment]
Awesome!

**Comment by ccordoba12** [comment]
Thank you man! I'm glad we were able to solve this one in conda-forge!! \o/ \o/

**Comment by jakirkham** [comment]
Appears this got stuck during conversion and needs a quick fix. I have proposed such a fix in this PR ( https://github.com/conda-forge/staged-recipes/pull/1072 ). Please let me know if it is acceptable.

**Pull request merged**